packages/template-retail-react-app/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salesforce/retail-react-app",
-  "version": "1.0.1",
+  "version": "2.0.0-preview.1",
```

Comment:
For retail-react-app, let's update the changelog, as Adam had done it in his PR: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1424/files#diff-3aec2f678f9138a6279a6c8b3ac8c3617136aef761dc310044a4e75487e75996